src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Services/FunctionsTestServer.cs

```diff
@@ -19,21 +18,25 @@ internal class FunctionsTestServer : IDisposable
         private readonly TestContext _context;
         private readonly Dictionary<string, string> hostConfig = new Dictionary<string, string>();
 
-        private readonly Dictionary<string, string> appConfig = new Dictionary<string, string>
-            {
-                { "EnvironmentName", "LOCAL" },
-                { "ConfigurationStorageConnectionString", "UseDevelopmentStorage=true" },
-                { "ConfigNames", "SFA.DAS.EmployerIncentives.Functions" },
-                { "NServiceBusConnectionString", "UseDevelopmentStorage=true" },
-                { "AzureWebJobsStorage", "UseDevelopmentStorage=true" }
-            };
+        private readonly Dictionary<string, string> appConfig;
 
         private IHost _host;
 
+        public SendInvitationRemindersHandler SendInvitationRemindersHandler { get; private set; }
+
         public FunctionsTestServer(TestContext context)
         {
             _isDisposed = false;
             _context = context;
+            appConfig = new Dictionary<string, string>
+            {
+                { "EnvironmentName", "LOCAL" },
+                { "ConfigurationStorageConnectionString", "UseDevelopmentStorage=true" },
+                { "ConfigNames", "SFA.DAS.EmployerIncentives.Functions" },
+                { "NServiceBusConnectionString", "UseDevelopmentStorage=true" },
+                { "AzureWebJobsStorage", "UseDevelopmentStorage=true" },
+                { "SendRemindersAfterThisNumberDays", _context.SendRemindersAfterThisNumberDays },
```

Comment:
Interesting, how does this work?  I'd have thought that `FunctionsTestServer` is constructed before the step that sets `SendRemindersAfterThisNumberDays` is run.

Reply:
Good question....

Reply:
You just found why one of my tests is failing. As you suspect this doesn't work